### PR TITLE
Catch up with recent Firedrake

### PR DESCRIPTION
**Pull request by mlange05**
Small housekeeping PR that makes everything run with latest Firedrake again and adds a pip install, versioning, a README and renames the base package to `seigen`. Following this merge we should be ready to tag a `v0.9-beta` release, from which to finalize the paper.

Welcome home, old friend...

### CODE REVIEW

**Comment by mlange05** [comment]
Also, very importantly, this merge now adds an MIT license, to finally make this open-source.

**Comment by FabioLuporini** [comment]
OK, go ahead.

File: LICENSE.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2017, Imperial College, London
```
Comment (ctjacobs) [inline LICENSE.md]: Remove the commas (i.e. "Copyright (c) 2017 Imperial College London") to be consistent with the "author" field in `setup.py` and the structure of the copyright line given [here](https://opensource.org/licenses/MIT).

**Comment by ctjacobs** [comment]
I think this is fine. I'd suggest removing `marmousi.py` and `elastic_wave/data` since these were never really used. Same goes for `tests/LOH.1` which I don't think ever worked.

It's also worth changing the repository's description to "A finite element-based elastic wave equation solver for seismological problems" so people don't get confused with other parts of the OPESCI project.

**Comment by mlange05** [comment]
> I think this is fine. I'd suggest removing marmousi.py and elastic_wave/data since these were never really used. Same goes for tests/LOH.1 which I don't think ever worked.

I actually meant to start playing with Marmousi in a little while, so I'd actually like to keep it for now. I'll also go through the test cases in some more detail soon with the paper overhaul, so again I'd like to keep them as they are for now. I'll also set up a travis builder for them so that we can rigorously go through what works and what doesn't at the point (eg. one of the `pulse` tests triggers some TSFC problems atm). So are you ok with postponing test cleanup for another PR?

**Comment by ctjacobs** [comment]
Yes that's fine.

**Pull request merged**